src/starcounter-debug-aid-ui.html

```diff
@@ -0,0 +1,111 @@
+﻿<link rel="import" href="puppet-js-view-model-tree.html" />
```

Comment:
There should be a import of "polymer.html" here as well. Every web component needs to import all of its dependencies, not assume that they are imported already.